docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-lambda-function.md

```diff
@@ -4,76 +4,71 @@ title: Collect Amazon CloudWatch Logs using a Lambda Function
-
-### Assign CloudWatch Log Groups to Your Lambda Function
-
-1. Go to the **Triggers** tab of your Lambda function.
-1. Select **Add Trigger.**
-1. In the **Add Trigger **prompt, click the box as instructed and select **CloudWatch Logs **from the dropdown menu.
-1. Select a** **CloudWatch Log Group to add to your function. You need at least one CloudWatch Log Group to see this option. For details on creating a log group, see [create a CloudWatch Log Group](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Create-Log-Group.html).
-1. Add a **Filter Name** to your trigger.
-1. (Optional) you can add a **Filter Pattern** to your trigger. For information about AWS filter patterns, see [Filter and Pattern Syntax](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html) in AWS documentation 
-1. Click **Enable Trigger**.
-1. Click **Submit** to add the trigger to your Lambda function.
+    * **Filter Pattern (Optional)**. May be left blank. For information about AWS filter patterns, see [Filter and Pattern Syntax](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html) in AWS help.
+1. In the **Environment Variables** section, set the value of the variable to the URL of the HTTP source from [Step 1](#step-1-add-a-hosted-collector-and-http-source) to which your logs will be sent. In addition, you can set any of the following optional variables:<br/><img src={useBaseUrl('img/send-data/environment-variables.png')} style={{border: '1px solid black'}} alt="environment-variables" width="700"/>
+    * **ENCODING (Optional)**. Use when decoding CloudWatch log events, default is **utf-1**.
+    * **SOURCE_CATEGORY_OVERRIDE (Optional)**. Override `_sourceCategory` value configured for the HTTP source.  
+    * **SOURCE_HOST_OVERRIDE (Optional)**. Override `_sourceHost` value configured for the HTTP source.
+    * **SOURCE_NAME_OVERRIDE (Optional)**. Override `_sourceName` value configured for the HTTP source.
+1. Click **Create Function.** A success message will be displayed.<br/><img src={useBaseUrl('img/send-data/success.png')} style={{border: '1px solid black'}} alt="success" width="700"/>
+
+## Step 3: Assign CloudWatch Log Groups to Your Lambda Function
+
+:::info
+You will need at least one CloudWatch Log Group to assign to your Lambda function. For details on how to create a CloudWatch Log Group, see [Create a CloudWatch Log Group](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Create-Log-Group.html).
+:::
 
 :::note
-If you have more than one CloudWatch Log Group to assign to your Lambda function, repeat the above steps for each Log Group.
+If you have more than one CloudWatch Log Group to assign to your Lambda function, repeat the below steps for each Log Group, else skip this step.
```

Comment:
```suggestion
If you have more than one CloudWatch Log Group to assign to your Lambda function, repeat the below steps for each Log Group. Otherwise, skip this step.
```